### ISSUE TITLE
Gambit TVL Showing as Zero

### ISSUE THREAD

**Issue by xvi10** [DefiLlama/DefiLlama-Adapters#201]
The latest TVL for Gambit shows zero even though there are assets locked in the queried contract at 0xc73A8DcAc88498FD4b4B1b2AaA37b0a2614Ff67B

<img width="1141" alt="Screenshot 2021-06-07 at 11 08 55 AM" src="https://user-images.githubusercontent.com/72239691/120953986-c7fe3880-c780-11eb-83f6-225cc4f3cb17.png">

**Comment by 0xngmi** [comment]
Sorry about that, the issue was that our RPC nodes failed and the adapter interpreted the failures as the tokens listed not being there and thus having a TVL of zero. I solved this back when it happened a few days ago and I'm currently working on improving our systems to reduce load.

In any case, thanks for notifying us, I didn't see this issue back then and that's why I haven't replied until now, but I took care of it (I have a system that announces me when there's irregular TVL changes).
